RevenueCatUI/CustomerCenter/Data/PurchaseInformation.swift

```diff
@@ -254,6 +262,18 @@ extension PurchaseInformation {
     }
 }
 
+extension PurchaseInformation: Identifiable {
+
+    var id: String {
+        let formatter = ISO8601DateFormatter()
+        let purchaseDateString = latestPurchaseDate.map { formatter.string(from: $0) }
+            ?? formatter.string(from: Date())
+        return "\(productIdentifier)_\(purchaseDateString)"
+    }
+}
```

Comment:
This felt like a good-enough id for now ??
I have another PR that adds a few useful variables to `PurchaseInformation`, that probably will replace `latestPurchaseDate`

Reply:
why not transaction id?